app/components/dynamic-content/dynamic-content-form/dynamic-content-form-field/dynamic-content-form-field-object/dynamic-content-form-field-object.js

```diff
@@ -33,9 +36,15 @@ angular.module('bulbs.cms.dynamicContent.form.field.object', [
           scope.$watch('schema', function () {
             if (_.has(scope.schema, 'fields')) {
               var fieldKeys = Object.keys(scope.schema.fields);
+              var includeOnlyProvided = _.isArray(scope.includeOnly);
+              var hideOnlyChildLabel = false;
 
-              if (_.isArray(scope.includeOnly)) {
+              if (includeOnlyProvided) {
                 fieldKeys = _.intersection(fieldKeys, scope.includeOnly);
+
+                if (scope.includeOnly.length === 1) {
+                  hideOnlyChildLabel = true;
```

Comment:
If there's only 1 item in the `include-only` array provided, don't show the label. This doesn't make a ton of sense, but it avoids a breaking change.

Optimally, at some point, I think we want to have `include-only` really be called `root-node` and have it be just one key instead of a list of keys. Definitely have a use for `root-node`, not so much `include-only`.